src/Select2/Select2.less

```
@@ -48,10 +62,22 @@
     .border--m(@primary_blue);
   }
 
+  &--error {
+    .border--m(@alert_red);
+  }
+
   /* stylelint-disable-next-line no-descending-specificity */
   &:hover {
     .border--m(@primary_blue);
   }
+
+  &--error:hover {
+    .border--m(@alert_red);
+  }
```

I'm not 100% sure, but I think you could combine this with the rule a few lines above like this.
```suggestion
  &--error, &--error:hover {
    .border--m(@alert_red);
  }
```

---

I tried using commas in alot of places and they didn't seem to work. Not sure if it's my css problem or a less quirk or something, but I couldn't get it to work

---

Sounds like I need to review selectors 🦧 . Carry on.